projects/js-toolkit/packages/liferay-cli/src/new/facet-plain-js/templates/src/index.js.ejs

```diff
@@ -7,35 +7,35 @@
  * @param  {Object} params a hash with values of interest to the portlet            
  * @return {void}                                                                   
  */                                                                                 
-export default function main(props) { 
-	const node = document.getElementById(props.portletElementId);
+export default function main(params) { 
```

Comment:
nice change here, it can seem insignificant at first but the name "props" carries so much weight these days that its helpful to read params and not immediately think "react"